docs/5.x/changelog.md

```diff
@@ -21,6 +21,24 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+### 5.2.0-rc.1
+
+#### Improvements
+
+* Introduce gravity terraform provider. See [Terraform Provider](/terraform) for details.
+* Refactor join operation to use FSM approach.
+* Do not show interface selection prompt in UI install flow if the installer machine has a single interface.
```

Comment:
`Suppress selection prompt in UI install flow if the installer machine has a single network interface.`